test/integration/apiserver/cel/validatingadmissionpolicy_test.go

```diff
@@ -210,7 +210,7 @@ func Test_ValidateNamespace_NoParams(t *testing.T) {
 					Name: "test-k8s",
 				},
 			},
-			err:           "namespaces \"test-k8s\" is forbidden: ValidatingAdmissionPolicy 'validate-namespace-suffix' with binding 'validate-namespace-suffix-binding' denied request: expression 'has(params.metadata) && has(params.metadata.name) && object.metadata.name.endsWith(params.metadata.name)' resulted in error: invalid type for field selection.",
+			err:           "namespaces \"test-k8s\" is forbidden: ValidatingAdmissionPolicy 'validate-namespace-suffix' with binding 'validate-namespace-suffix-binding' denied request: failed expression: has(params.metadata) && has(params.metadata.name) && object.metadata.name.endsWith(params.metadata.name)",
```

Comment:
Curious why it was passed earlier...

Reply:
CEL changed how this error message is formatted.

Reply:
Looks like not a cel changes since it didn't change in : https://github.com/kubernetes/kubernetes/blob/68901de8981a0908d7cfdba0d9c19aeb50385f35/test/integration/apiserver/cel/validatingadmissionpolicy_test.go#L181